index.html

```diff
@@ -1034,60 +1034,95 @@
+APIs</a>.
+</aside>
 
-<div class="practice">
+In maintaining duties of <a href="#dfn-duty-of-protection">protection</a>, <a
+href="#dfn-duty-of-discretion">discretion</a> and <a href="#dfn-duty-of-loyalty">loyalty</a>, user
+agents should be conservative when sharing data not necessary to satisfy a user's immediate goals.
+
+On the Web, data is often shared even outside the need to load and display a page for <dfn>ancillary
+uses</dfn> including: 
+
+* browser telemetry, including: 
+  * performance measurements, 
+  * measuring feature usage, 
+  * debugging site, browser and networking issues, and, 
+  * detecting security problems or attacks; 
+* site telemetry, including: 
+  * performance measurements, 
+  * analytics about usage and visitors, 
+  * debugging site issues, and, 
+  * detecting security problems or attacks; 
+* advertising and social media sharing, including: 
+  * measurement, targeting, ad auctions, personalization, fraud prevention; 
+* performance, including caching and prefetching content; 
+* software updates.
+
+In some cases, mechanisms for sharing data are provided in order to minimize some other data
+collection mechanism identified as more intrusive or costly.
+
+For some uses, some people would reasonably anticipate sharing such data or identify that use as a
+commonly-accepted contextual norm and some people would be willing to participate (for example, might
+have explicitly decided to contribute if they had the time and understanding to consider details about
+such a use) and some people would identify those uses as supporting their own goals. 
+
+Aggregated or <a href="#dfn-de-identified">de-identified</a> data is often more likely to be
+acceptable in these senses and often still contributes to the collective benefit while minimizing
+privacy threats to a particular individual (see <a href="#principle-collective-privacy">collective
+privacy</a>). But some users might also object, in particular instances or in general.
+
+<span class="practicelab">Sites and user agents should seek to understand and respect people's goals
+and preferences about use of data about them.</span>
+
+Agents should both be conservative in applying data minimization to any <a
+href="#dfn-ancillary-uses">ancillary use</a> and also not burden the user with additional [=privacy
+labor=]. To that end, user agents may employ user research, solicitation of general preferences, and
+heuristics about sensitivity of data or trust in a particular context. To facilitate site
+understanding of user preferences, user agents can provide browser-configurable signals to directly
+communicate common user preferences.
+
+<aside class="example">
+  Sites and browsers wish to collect telemetry data to determine how frequently features are used or
+  to debug breakages, but the user agent does not want to burden the user with frequent consent
+  requests. A browser could use a first-run dialog to ask the user whether they generally support
```

Comment:
A first-run dialog for the browser installation right? Or is this for first visit on a given website?

Reply:
Ah, yes, I meant "first-run" to imply first run of a browser or piece of local software. But if that's not clear, we should wordsmith it.

And clearly this is just an example. Maybe user agents should check in with their users occasionally, rather than asking a bunch of questions on first-run/installation. We should encourage experimentation.